.buildkite/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+common-params:
+  &docker-container
+  docker#v3.8.0:
+    image: "public.ecr.aws/automattic/android-build-image:v1.1.0"
+    propagate-environment: true
+    environment:
+      # DO NOT MANUALLY SET THESE VALUES!
+      # They are passed from the Buildkite agent to the Docker container
+      - "AWS_ACCESS_KEY"
+      - "AWS_SECRET_KEY"
+
+steps:
+  - label: "Lint & Checkstyle"
+    key: "lint_and_checkstyle"
+    plugins:
+      - *docker-container
+    command: |
+      ./gradlew lintRelease checkstyle
+    artifact_paths:
+      - "**/build/reports/lint-results**.*"
+      - "**/build/reports/checkstyle/checkstyle.*"
+
+  - label: "Test"
+    key: "test"
+    plugins:
+      - *docker-container
+    command: |
+      ./gradlew test
```

Comment:
I don't think we have tests in the repo. Unless you want to leave it for future and to have all pipelines standardized

Reply:
Thank you for pointing it out, I'll remove this step.

Reply:
Addressed in 3b48de41153c6f0305d87dc20a6ff58ce60fe613.